testing/tests/DevExpress.ui.widgets.dataGrid/keyboardNavigation.keyboardKeys.tests.js

```diff
@@ -1708,7 +1708,7 @@ QUnit.module('Keyboard keys', {
         // act
         this.gridView.render($('#container'));
         this.editingController.editCell(0, 1);
-        this.keyboardNavigationController._focusedCellPosition = { columnIndex: -1, rowIndex: -1 };
+        this.keyboardNavigationController._focusedCellPosition = { columnIndex: 0, rowIndex: 1 };
```

Comment:
it is strange that after editing _focusedCellPosition is set to (-1, -1) in this test. There are couple another tests for T259896 below this test, where _focusedCellPosition is set to edited cell position. Please check this.